indigo_api/templates/indigo_api/akn/coverpage_act.html

```diff
@@ -55,27 +55,124 @@ <h2>{{ subt }}</h2>
+        {% with document.work.main_commencement as commencement %}
+        {% with commencement.commencing_work as commencing_work %}
+        {% if not commencement.date %}
+          <li class="commencement-date">
+            {% blocktrans %}Commencement date unknown{% endblocktrans %}
+              {% if commencing_work %}
+                {% with numbered_title=commencing_work.numbered_title title=commencing_work.title %}
+                  {% work_resolver_url commencing_work as resolver_uri %}
+                  {% if numbered_title %}
+                    {% blocktrans %}– commenced by <a href="{{ resolver_uri }}">{{ numbered_title }}</a>{% endblocktrans %}
+                  {% else %}
+                    {% blocktrans %}– commenced by <a href="{{ resolver_uri }}">{{ title }}</a>{% endblocktrans %}
+                  {% endif %}
+                {% endwith %}
+              {% endif %}
+          </li>
+        {% else %}
+          <li class="commencement-date">
+            {% blocktrans with date=commencement.date|date:"j E Y" %}Commenced on {{ date }}{% endblocktrans %}
+              {% if commencing_work %}
+                {% with numbered_title=commencing_work.numbered_title title=commencing_work.title %}
+                  {% work_resolver_url commencing_work as resolver_uri %}
+                  {% if numbered_title %}
+                    {% blocktrans %}by <a href="{{ resolver_uri }}">{{ numbered_title }}</a>{% endblocktrans %}
+                  {% else %}
+                    {% blocktrans %}by <a href="{{ resolver_uri }}">{{ title }}</a>{% endblocktrans %}
+                  {% endif %}
+                {% endwith %}
+              {% endif %}
+          </li>
+        {% endif %}
+        {% endwith %}
+        {% endwith %}
+      {% else %}
+        {% with document.work.commencements.all as commencements %}
         <li class="commencement-date">
-          {% blocktrans %}Commencement date unknown{% endblocktrans %}
+          {% blocktrans %}There are multiple commencements:{% endblocktrans %}
         </li>
-      {% elif document.commencement_date %}
-        <li class="commencement-date">
-          {% blocktrans with date=document.commencement_date|date:"j E Y" %}Commences on {{ date }} unless otherwise noted{% endblocktrans %}
-          {% with document.work.commencing_work as commencing_work %}
-            {% if commencing_work %}
-              {% with numbered_title=commencing_work.numbered_title title=commencing_work.title %}
-                <br>
-                {% work_resolver_url commencing_work as resolver_uri %}
-                {% if numbered_title %}
-                  {% blocktrans %}(see <a href="{{ resolver_uri }}">{{ numbered_title }} – {{ title }}</a>){% endblocktrans %}
+        {% if document.work.commencements_count < 3 %}
+          {% for commencement in commencements %}
+              {% commenced_provisions_description document commencement as provisions_description %}
```

Comment:
over-indented

Reply:
we'll probably want to pass in the work, not the document

Reply:
I had thought the document might be useful because it has a language, but if we're using the ToC there's not much we can do in any case, right?

Reply:
yeah, commenceable provisions works that out.